py/openage/convert/mediafile.py

```diff
@@ -77,6 +136,10 @@ def media_convert(args):
         args.extract.append('*:*.*')
 
     extraction_rules = [ExtractionRule(e) for e in args.extract]
+    if args.jobs == 0:
+        args.jobs = os.cpu_count()
```

Comment:
kay, then just leave it for now, although i don't see the problem with `import os` :)